app/controllers/concerns/application_controller/request_preprocessing.rb

```diff
@@ -72,8 +72,10 @@ def log_full_url
     Rails.logger.info("Request URL: #{request.url}")
   end
 
+  # deletable
   def set_default_nav_context
     @context = {}
+    nav_link_builder.context = {}
   end
```

Comment:
I would go ahead and delete this.

Reply:
I think you mean the line beginning @context, yes?  that's done.